[RM Ch2] To pass Action fail

petorre opened the pull request:
To pass action fail https://github.com/anuket-project/RM/actions/runs/11930908117/job/33252646132 . If this table fix works, I'll check where else it fails. If this table fix doesn't work, I'll then rework all tables to new format.

petorre committed:
Update chapter02.rst

To pass action fail https://github.com/anuket-project/RM/actions/runs/11930908117/job/33252646132

petorre committed:
Table 2-1 in new format

CsatariGergely committed:
Fix build errors

- Ignoring non working link
- Fixing references
- Remving debug logs from togsma.py

Signed-off-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Tables 3-6, 3-9 and 3-10

fixed formatting of 3-6 and 3-9
added number for table 3-10 and fixed formatting
changed number for previous 3-10 to 3-11

petorre committed:
unroll Table 3-9

previous commit results in rm.docx that cannot be opened by Word

petorre committed:
unroll Table 3-9

previous commit was to unroll Table 3-10
this is to unroll Table 3-9

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -796,28 +796,34 @@ below.
+   * - Management
+     - ``SFC orchestrator``
+     - High Level of orchestrator Orchestrate the SFC based on SFC Models/Policies with help of control components.
+   * - 
```
```suggestion
   * - Management
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -796,28 +796,34 @@ below.
+   * - 
+     - ``SFC OAM Components``
+     - Responsible for SFC OAM functions
+   * - 
```
```suggestion
   * - Management
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -796,28 +796,34 @@ below.
+   * - 
+     - ``VNF MANO``
+     - NFVO, VNFM, and VIM Responsible for SFC Data components lifecycle
+   * - 
```
```suggestion
   * - Management
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -796,28 +796,34 @@ below.
+   * - Control
+     - ``SFC SDN Controller``
+     - SDNC responsible to create the service specific overlay network. Deploy different techniques to stitch the wiring but provide the same functionality, for example l2xconn, SRv6, Segment routing etc.
+   * - 
```
```suggestion
   * - Control
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -1302,34 +1308,166 @@ Where:
+     - O
+     - O
+     - O
+   * - 
```
```suggestion
   * - Data-centre Storage
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -1302,34 +1308,166 @@ Where:
+     - O
+     - O
+     - O
+   * - 
```
```suggestion
   * - Data-centre Storage
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -1302,34 +1308,166 @@ Where:
+     - O
+     - O
+     - O
+   * - 
```
```suggestion
   * - Edge Cloud
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -1302,34 +1308,166 @@ Where:
+     - O
+     - O
+     - Y
+   * - 
```
```suggestion
   * - Edge Cloud
```

CsatariGergely commented on doc/ref_model/chapters/chapter03.rst:
```diff
@@ -1302,34 +1308,166 @@ Where:
+     - O
+     - O
+     - O
+   * - 
```
```suggestion
   * - Split Control/User Plane Edge Cloud
```

petorre committed:
delete list-table before Table 3-9

./chapters/chapter03.rst:1311: D000 The "list-table" directive is empty; content required.

petorre committed:
Table 3-9 with header-rows = 2

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Update doc/ref_model/chapters/chapter03.rst

Co-authored-by: Gergely Csatari <gergely.csatari@nokia.com>

petorre committed:
Table 3-9 without empty cells in first two rows

petorre committed:
fixed Table 3-10

petorre committed:
fixed Table 4-8 and renumbered Tables 4-*

petorre committed:
fixed Table 6-2 and 6-3

petorre committed:
fixed Table 7-10 and 7-16

petorre committed:
renumbered and fixed Tables 8-*

petorre committed:
fixed Table 8-9

petorre committed:
fixed Table 9-2

petorre committed:
fixed Table 7-3

petorre commented:
Now it seems to builds .docx well.

walterkozlowski reviewed (approved):
Approved after reviewing in the RM call.
